lms/static/certificates/sass/_build.scss

```diff
@@ -1,45 +1,53 @@
 // ------------------------------
 // Open edX Certificates: Shared Build Compile
 
-@import '../vendor/normalize/normalize';
+// About: Sass compile for Open edX Certificates elements that are shared between LTR and RTL UI. Configuration and vendor specific imports happen before this shared set of imports are compiled in the main-*.scss files.
 
 // ------------------------------
 // #UTILITIES
 // ------------------------------
-@import 'utilities/mixins';
+@import '../../../../static/edx-pl/sass/utilities/variables';
+@import '../../../../static/edx-pl/sass/utilities/functions';
+@import '../../../../static/edx-pl/sass/utilities/mixins';
+@import '../../../../static/edx-pl/sass/utilities/helpers';
+
+@import 'utilities.scss';
+
+// ------------------------------
+// #VENDOR
+// ------------------------------
+@import '../vendor/fontawesome/sass/font-awesome.scss';
 
 // ------------------------------
 // #BASE
 // ------------------------------
-@import 'base/fonts';
-@import 'base/graphics';
-@import 'base/anims';
-@import 'base/typography';
-@import 'base/reset';
+@import '../../../../static/edx-pl/sass/base/normalize';
+@import '../../../../static/edx-pl/sass/base/reset';
+@import '../../../../static/edx-pl/sass/base/typography';
+
+@import 'base.scss';
 
 // ------------------------------
 // #COMPONENTS
 // ------------------------------
-@import 'components/controls';
-@import 'components/navigation';
-@import 'components/header';
-@import 'components/footer';
+@import '../../../../static/edx-pl/sass/components/colors';
+@import '../../../../static/edx-pl/sass/components/buttons';
+@import '../../../../static/edx-pl/sass/components/headings';
+@import '../../../../static/edx-pl/sass/components/copy';
+
+@import 'components.scss';
 
 // ------------------------------
-// #VIEWS
+// #LAYOUTS
 // ------------------------------
-@import 'views/certificate';
-@import 'views/valid';
-@import 'views/invalid';
-@import 'views/validate';
+@import 'layouts';
 
 // ------------------------------
-// #CONTEXTS
+// #VIEWS
 // ------------------------------
-@import 'contexts/print';
-@import 'contexts/ie';
+@import 'views';
 
 // ------------------------------
 // #SHAME
 // ------------------------------
-@import 'utilities/shame';
+@import 'overrides';
```

Comment:
NOTE: overrides contains shame, developer, and other last of the compile extensions.